src/utils/validator_state.py

```diff
@@ -2,7 +2,7 @@
     MAX_EFFECTIVE_BALANCE,
     ETH1_ADDRESS_WITHDRAWAL_PREFIX,
     SHARD_COMMITTEE_PERIOD,
-    FAR_FUTURE_EPOCH,
+    FAR_FUTURE_EPOCH, EFFECTIVE_BALANCE_INCREMENT,
```

Comment:
split in two lines

Reply:
ebad50a2374017e6b76d650376627ec8f902e9cb